pkg/kubelet/rkt/rkt.go

```diff
@@ -1139,7 +1175,16 @@ func netnsPathFromName(netnsName string) string {
 // configured NetworkPlugin's setup function on it.
 // It returns the namespace name, configured IP (if available), and an error if
 // one occured.
+//
+// If the pod is running in host network or is running using the no-op plugin, then nothing will be done.
 func (r *Runtime) setupPodNetwork(pod *api.Pod) (string, string, error) {
+	glog.V(3).Infof("Calling network plugin %s to set up pod for %s", r.networkPlugin.Name(), format.Pod(pod))
```

Comment:
nit, this log might make more sense after the noop check. Same for cleanup above.